tests/test_multistore_file.py

```diff
@@ -27,6 +34,32 @@ def filename(self):
 
 
 class MultistoreFileTests(unittest.TestCase):
+    def tearDown(self):
```

Comment:
No, I don't think it's a thing, I think the style guides leave it ambiguous and my formatter only works one way :-\ I auto-format on save and I can't figure out how to get Intellij not to do it. There is an issue on Jetbrains to let you configure it, I should just diasble auto-format for this project.